tests/rt.conf

```diff
@@ -433,3 +433,6 @@ RUN | datm_cdeps_control_cfsr                           | + hera hercules
 ### ATM-FBH test ###
 COMPILE | atm_fbh | intel | -DAPP=ATMF -DCCPP_SUITES=FV3_HRRR -D32BIT=ON | - wcoss2  noaacloud  acorn | fv3 |
 RUN | cpld_regional_atm_fbh                             | - wcoss2  noaacloud  acorn           | baseline |
+
+### UFS with MPAS dynamical core ###
+COMPILE | atm_mpas_dyn32 | intel | -DAPP=ATMMPAS -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v17_p8_ugwpv1 -D32BIT=ON | | mpas |
```

Comment:
I do not see RUN step that executes the ufs-model executable built with this COMPILE step. How do we even know the mpas model is compiled correctly. Do you plan to add at least one regression test for ufs with mpas core?

Reply:
Okay, I see this question has been already asked in one of the previous comments, and the answer was no. I'm now confused, but I do not see what we get by making these changes in the ufs-weather-model at this time.  We can discuss the corresponding PR in the fv3atm and if it's necessary merge it, but why updating ufs-wm without any tests is necessary

Reply:
`-D32BIT=ON` is build flag used to build fv3 dycore using single-precision reals as a default. Is it used in MPAS core? I do not see it used in any mpas CMakeLists.txt. If it's not used, it should be removed from this COMPILE step.